LoggerFinal/src/main/java/com/Logger/utils/ParserUtility.java

```diff
@@ -0,0 +1,135 @@
+package com.Logger.utils;
+
+import com.Logger.domain.model.*;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Created by technology on 19/4/17.
+ */
+@Service
+public class ParserUtility {
+    @Autowired
+    Parser parser;
+
+    List<Inventory> inventoryList;
+
+    List<Price> priceList;
+
+    public List<Inventory> createInventoryObjectNode(List<ObjectNode> inventoryNode) {
+        inventoryList = new LinkedList<>();
+
+        for (ObjectNode node : inventoryNode) {
+            Inventory inventory = new Inventory();
+
+            inventory.createdDate = parser.getCurrentDate();
+            inventory.uniqueId = parser.getLong("uniqueId", node);
+            //inventory.status = parser.getStatus("status", node);
+            inventory.reservationId = parser.getLong("reservationId", node);
+            inventory.revisionId = parser.getLong("revisionId", node);
+            inventory.type = parser.getTypeValue("revisionId", "reservationId", node);
+
+            ArrayNode hotelsNode = (ArrayNode) node.get("hotels");
+            List<Hotels> hotels = new LinkedList<>();
+            for (JsonNode hotelNode : hotelsNode) {
+                Hotels hotel = new Hotels();
+                hotel.hotelId = hotelNode.get("hotelId").intValue();
+
+                List<Rooms> rooms = new LinkedList<>();
+                ArrayNode roomsNode = (ArrayNode) hotelNode.get("rooms");
+                for (JsonNode roomNode : roomsNode) {
+                    Rooms room = new Rooms();
+                    room.roomId = roomNode.get("roomId").intValue();
+
+                    List<Availability> availabilities = new LinkedList<>();
+                    ArrayNode availabilitiesNode = (ArrayNode) roomNode.get("availability");
+                    for (JsonNode availabilityNode : availabilitiesNode) {
+                        Availability availability = new Availability();
+                        availability.free = availabilityNode.get("free").intValue();
+                        availability.date = parser.getDate("date", availabilityNode);
+                        availabilities.add(availability);
+                    }
+                    rooms.add(room);
+                    room.availability = availabilities;
+                }
+                hotel.rooms = rooms;
+                hotels.add(hotel);
+                inventory.hotels = hotels;
+            }
+            inventoryList.add(inventory);
+        }
+        return inventoryList;
+    }
+
+    public List<Price> createPriceObjectNode(List<ObjectNode> priceNode) {
```

Comment:
ideal function should not be more than 5-10 lines of code. Break down this function into smaller functions. This way code becomes more readable and easy to debug